src/parser/shared/modules/features/Checklist/Rule.js

```diff
@@ -94,6 +102,13 @@ class Rule extends React.PureComponent {
 
     const performance = this.performance;
     const passed = this.passed;
+    const requirements = this.props.children;
+    console.log(this.props);
+    console.log(requirements);
```

Comment:
looks like you left some stray `console.log`s in. other than that it looks good

Reply:
woops, should be removed now.